src/Raven.Server/RavenServer.cs

```diff
@@ -1298,6 +1298,11 @@ internal void SetCertificate(X509Certificate2 certificate, byte[] rawBytes, stri
                 return true;
             }
 
+            if (tcp.Operation == TcpConnectionHeaderMessage.OperationTypes.TestConnection)
+            {
+                return true;
```

Comment:
Huh? This is for a specific database, but you are returning true for server wide?